playbooks/aws/openshift-cluster/vars.yml

```diff
@@ -31,3 +31,100 @@ deployment_vars:
   enterprise: "{{ deployment_rhel7_ent_base }}"
   openshift-enterprise: "{{ deployment_rhel7_ent_base }}"
   atomic-enterprise: "{{ deployment_rhel7_ent_base }}"
+
+provision:
+  cert_arn: "arn:aws:iam::000000000000:server-certificate/<clusterid>_<domain>"
+  iam_cert_ca:
+    name: test_openshift
+    cert_path: '/path/to/wildcard.<clusterid>.example.com.crt'
+    key_path: '/path/to/wildcard.<clusterid>.example.com.key'
+    chain_path: '/path/to/cert.ca.crt'
+  yum_repositories:
```

Comment:
Yeah, I'm not against name spacing things and I actually like nested dictionaries for a lot of things.  We should just break out the nest dictionaries to refer back to other variables.  IE:

```
provision_yum_repos: ... some dict ...

provision:
  yum_repositories:  {{ provision_yum_repos }}
```

This will give us maximum flexibility and allow users to override just portions of the larger dictionary.